common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -475,3 +475,36 @@ def test_cc_license(self):
         # The course_license text will include a bunch of screen reader text to explain
         # the selected options
         self.assertIn("Some Rights Reserved", self.lms_courseware.course_license)
+
+
+@attr('a11y')
+class StudioSettingsA11yTest(StudioCourseTest):
```

Comment:
Thank you for adding this test! Do we run the Bok Choy tests with all the new fields visible (ENABLE_EXTENDED_COURSE_DETAILS feature flag enabled).